src/test/java/it/pagopa/pn/deliverypush/middleware/externalclient/pnclient/datavault/PnDataVaultClientReactiveImplTest.java

```diff
@@ -0,0 +1,106 @@
+package it.pagopa.pn.deliverypush.middleware.externalclient.pnclient.datavault;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.pagopa.pn.commons.exceptions.PnInternalException;
+import it.pagopa.pn.datavault.generated.openapi.clients.datavault.model.BaseRecipientDto;
+import it.pagopa.pn.datavault.generated.openapi.clients.datavault.model.RecipientType;
+import it.pagopa.pn.deliverypush.LocalStackTestConfig;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockserver.client.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.MediaType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import reactor.core.publisher.Flux;
+
+import java.util.List;
+
+import static org.mockserver.integration.ClientAndServer.startClientAndServer;
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+@SpringBootTest
+@ActiveProfiles("test")
+@TestPropertySource(properties = {
+        "pn.delivery-push.data-vault-base-url=http://localhost:9998",
+})
+@Import(LocalStackTestConfig.class)
```

Comment:
Essendo un test di integrazione, mettiamogli il suffisso IT (i test di integarzione vengono eeseguiti nella fase di very se la classe di test ha il suffisso IT (e non solo).